ncm-network/src/main/perl/nmstate.pm

```diff
@@ -583,6 +610,21 @@ sub Configure
         $self->ethtool_opts_keeps_state($file_name, $ifacename, $iface, $exifiles);
     }
 
+    # configure vips defined under path /system/network/vips/ as dummy interface
+    # only if manage_vips is set to true.
+    my $vips = $net->{vips};
```

Comment:
It's been a while since I wrote Perl, does it warn if a key is missing? I would have thought changing the order here would result in less chance of this emitting spurious warnings:

```
if ($net->{manage_vips} && defined($net->{vips})) {
  my $vips = $net->{vips};
  $self->verbose("Service address vips found, configuring dummy interfaces");
```

Reply:
doesn't give a WARN, and an empty string in perl is considered to be false from what I read, AFAIK.